app/views/groups/users.html.erb

```diff
@@ -1,14 +1,26 @@
+<div class="sml-title-purple-align-right">
+  directory
+</div>
+
+<div class="two-color-sentence-align-right">
+  <%= @group.name.downcase %>
+</div>
 
-<h1>Welcome to <%= @group.name %></h1>
-<div class="mentor-list">
+<div class="group-list">
   <% @users.each do |user| %>
-    <div class="mentor-card">
```

Comment:
Same applies here for changing the class name from mentor-card to parent-card. The class name in the scss file will need to be changed too. I may have overlooked it though and you may've changed it already. No biggie whatsover.